src/Gizmos/lightGizmo.ts

```diff
@@ -39,6 +47,19 @@ export class LightGizmo extends Gizmo {
         this._material = new StandardMaterial("light", this.gizmoLayer.utilityLayerScene);
         this._material.diffuseColor = new Color3(0.5, 0.5, 0.5);
         this._material.specularColor = new Color3(0.1, 0.1, 0.1);
+
+        if (gizmoLayer) {
+            this._pointerObserver = gizmoLayer.utilityLayerScene.onPointerObservable.add((pointerInfo) => {
+                if (!this._light) {
+                    return;
+                }
+
+                var isHovered = pointerInfo.pickInfo && (this._rootMesh.getChildMeshes().indexOf(<Mesh>pointerInfo.pickInfo.pickedMesh) != -1);
+                if (isHovered && pointerInfo.type === PointerEventTypes.POINTERDOWN && pointerInfo.event.button === 0) {

```

Comment:
If this should be executed only on pointer down you can prefilter using the second variable of the `add` function:

```javascript
gizmoLayer.utilityLayerScene.onPointerObservable.add((pointerInfo) => {
    /* guaranteed to be a pointer down event*/
}, PointerEventType.POINTERDOWN);
```

Reply:
Fixed